packages/core/src/components/control-card/controlCard.tsx

```diff
@@ -91,7 +91,7 @@ export const ControlCard: React.FC<ControlCardProps> = React.forwardRef((props,
 
     // use a container element to achieve a good flex layout
     const labelElement = <div className={Classes.CONTROL_CARD_LABEL}>{children ?? label}</div>;
-    const controlProps: ControlProps = {
+    const controlProps: Omit<ControlProps, "size"> = {
```

Comment:
This type feels used often enough that I think it would be worth naming it so that consumers can interact with it directly, rather than having to type out the clunk `Omit` type or the props index type (e.g. `ControlProps["size"]`). The only issue is what to call it? I think something simple like `NonSmallSize` is probably sufficient, but definitely open to bikeshedding on this.

Reply:
@ggdouglas Any further thoughts on this?

Reply:
Not sure I quite understand what you mean. This particular `Omit` is needed due to [`size`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/size) being a valid attribute on HTML input, which comes in conflict with the `"size"` attribute that we are adding to controls. Without it, we get the following compilation error:
```
Types of property 'size' are incompatible.
    Type 'number | undefined' is not assignable to type 'Omit<Size, "small"> | undefined'.
      Type 'number' is not assignable to type 'Omit<Size, "small">'.
```

Reply:
0fc046db9dbbc3afeaa799fb4e59f12c32241cf3